demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -221,6 +226,11 @@ export class DemoMeetingApp
   static readonly DATA_MESSAGE_TOPIC: string = 'chat';
   static readonly DATA_MESSAGE_LIFETIME_MS: number = 300000;
 
+
+  // ideally we don't need to change this. Keep this configurable in case users have super slow network.
+  loadingBodyPixDependencyTimeoutMs: number = 10000;
+  loadingBodyPixDependencyPromise: Promise<void>;
```

Comment:
Strictly speaking, this should be `: undefined | Promise<void>;`. At some point we will turn on strict mode and this will be an error.

Reply:
thanks ! will fix